src/commands/indexer/index.ts

```diff
@@ -223,13 +223,22 @@ export default class Indexer extends HealthCheck {
 
   processDBJobs(timestamp?: number, job?: DBJob): void {
     if (timestamp !== undefined && job !== undefined) {
```

Comment:
I think we should have a log here before the IF condition to make sure its actually getting into this function